scheme/data.py

```diff
@@ -0,0 +1,153 @@
+from faker import Faker
+from decimal import Decimal
+import random as r
+import json
+
+
+f = Faker()
+Faker.seed(0)  # we are using the same data all the time here
+r.seed(0)
+lat = 37.773972
+lng = -122.431297
+
+
+def genId():
+    return f.md5()
+
+
+def pickOne(lst):
+    return lst[r.randint(0, len(lst)-1)]
+
+
+def location():
+    return dict(
+        address=f.address(),
+        lat=lat + float(f.latitude())/1000,
```

Comment:
minor: lat += float(f.latitude()) / 1000

Reply:
man, i forgot how to python hahahaha

Reply:
this is inside the dict, so not feasible hahaha